### PR TITLE
add material darker theme

### PR DESCRIPTION
**Pull request by czhang03**

### CODE REVIEW

File: material-darker.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+		<key name=".Vanilla" modified="2017-02-15 13:30:28" build="160724">
```
Comment (shiitake) [inline material-darker.xml:1]: Instead of .Vanilla the name value should be Palette1

File: material-darker.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			<value name="ColorTable00" type="dword" data="00151515"/>
```
Comment (shiitake) [inline material-darker.xml]: It looks like the theme is missing several common elements. 

Particularly Name, ExtendedColors, ExtendColorIdx, TextColorIdx, BackColorIdx, PopTextColorIdx, PopBackColorIdx. 

You might compare this to another one of the themes to verify - [moe-dark](https://github.com/joonro/ConEmu-Color-Themes/blob/master/moe-dark.xml)

**Review by shiitake — CHANGES_REQUESTED** [review]
I'm really excited to try this theme out! It looks very cool.   :)

Please see a few of the recommended changes and let me know if you have any questions.

File: material-darker.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+			<value name="ColorTable22" type="dword" data="00008080"/>
+			<value name="ColorTable23" type="dword" data="00c0c0c0"/>
+			<value name="ColorTable24" type="dword" data="00808080"/>
+			<value name="ColorTable25" type="dword" data="00ff0000"/>
+			<value name="ColorTable26" type="dword" data="0000ff00"/>
+			<value name="ColorTable27" type="dword" data="00ffff00"/>
+			<value name="ColorTable28" type="dword" data="000000ff"/>
+			<value name="ColorTable29" type="dword" data="00ff00ff"/>
+			<value name="ColorTable30" type="dword" data="0000ffff"/>
+			<value name="ColorTable31" type="dword" data="00ffffff"/>
```
Comment (shiitake) [inline material-darker.xml:33]: It looks like the theme is missing several common elements.

Particularly Name, ExtendedColors, ExtendColorIdx, TextColorIdx, BackColorIdx, PopTextColorIdx, PopBackColorIdx.

You might compare this to another one of the themes to verify - moe-dark

**Comment by czhang03** [comment]
thank you, I will have it fixed soon

**Comment by joonro** [comment]
Hi - I merged your changes and completed the update of the repo - please see `material-sublime-darker` for your color theme.

**Comment by czhang03** [comment]
Thank you. 

Sorry this has been so long, I almost forget about this pr.

Thank you again

**Pull request merged**